src/libraries/Common/src/Microsoft/Win32/SafeHandles/GssSafeHandles.PlatformNotSupported.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    internal sealed class SafeGssNameHandle : SafeHandle
```

Comment:
Where are the handled used? Can we fix the refereces if they are not usable?

Reply:
Last time I checked they were leaked from the PAL abstraction all over the place so it's not exactly easy to remove them.

Reply:
If they are used as type then it's fine but it should not be constructed (e.g. we can make the ctor private instead of PNSE)

Reply:
Made the ctors private, thanks